core/inject.js

```diff
@@ -77,7 +77,7 @@ Blockly.inject = function(container, opt_options) {
 
   var workspace = Blockly.createMainWorkspace_(svg, options, blockDragSurface,
       workspaceDragSurface);
-  Blockly.user.keyMap.setKeyMap(options.keyMap);
+  Blockly.user.keyMap.setKeyMap(workspace.options.keyMap);
```

Comment:
Why not continue using options? workspace.options should be == options.

Reply:
Changed back.